Makefile

```diff
@@ -6,7 +10,11 @@ SUB_DIRS = utils/fabric-ccenv-sgx ercc ecc_enclave ecc tlcc_enclave tlcc
 all build test clean :
 	$(foreach DIR, $(SUB_DIRS), $(MAKE) -C $(DIR) $@;)
 
-checks: linter
+checks: license linter
+
+license:
+	@echo "License: Running licence checks.."
+	@sh ${GOPATH}/src/github.com/hyperledger/fabric/scripts/check_license.sh
```

Comment:
Calling it via ${TOP}/../../hyperledger/fabric/scripts/check_license.sh seem a bit more robust (i.e., it will also work when you have multiple GOPATH components)?  But i have no strong opinion on this ...